File: 07_scripts/ops/clear_dashboard_storage.js
const { chromium } = require('playwright');

const URL = process.env.TARGET_URL || 'http://host.docker.internal:18789/';

(async () => {
  const browser = await chromium.launch({ headless: true });
  const context = await browser.newContext();
  const page = await context.newPage();
  await page.goto(URL, { waitUntil: 'networkidle' });
  await page.evaluate(() => { localStorage.clear(); sessionStorage.clear(); });
  await context.clearCookies();
  await page.reload({ waitUntil: 'networkidle' });
  console.log('Storage cleared and page reloaded:', page.url());
  await browser.close();
})();

File: 07_scripts/ops/clear_dashboard_storage.py
from playwright.sync_api import sync_playwright

URL = "http://127.0.0.1:18789/"

def main():
    with sync_playwright() as p:
        browser = p.chromium.launch(headless=True)
        context = browser.new_context()
        page = context.new_page()
        page.goto(URL, wait_until='networkidle')
        page.evaluate("localStorage.clear(); sessionStorage.clear();")
        context.clear_cookies()
        page.reload(wait_until='networkidle')
        print(f"Storage cleared and page reloaded: {page.url}")
        browser.close()

if __name__ == '__main__':
    main()
